app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -157,29 +195,31 @@
     "Solomon Islands",
     "Somalia",
     "South Africa",
+    "South Georgia and the South Sandwich Islands",
     "South Korea",
     "South Sudan",
     "Spain",
     "Sri Lanka",
-    "Saint Vincent and the Grenadines",
-    "Palestine",
+    "Svalbard and Jan Mayen",
     "Sudan",
     "Suriname",
     "Swaziland",
     "Sweden",
     "Switzerland",
     "Syria",
+    "Tahiti",
     "Taiwan",
     "Tajikistan",
     "Tanzania",
-    "Macedonia",
     "Thailand",
     "Togo",
+    "Tokelau",
     "Tonga",
     "Trinidad and Tobago",
     "Tunisia",
     "Turkey",
     "Turkmenistan",
+    "Turks and Caicos Islands",
     "Tuvalu",
     "United Kingdom",
     "United States",
```

Comment:
The next line is "Unknown", is this because there are countries which haven't been discovered yet, or should it say "Not Listed" instead? ;)

(Sorry, British pedant over here ;) )